### PR TITLE
Update themes.rst - Addition to documentation

### PR DESCRIPTION
**Pull request by patsuckow**
You can add to the documentation to clarify.

### CODE REVIEW

**Comment by patsuckow** [comment]
That is, I want to say that you can add this item to clarify.

File: en/views/themes.rst [hunk 1]
```diff
@@ -43,6 +43,14 @@ If a view file can't be found in the theme, CakePHP will try to locate the view
 file in the **src/Template/** folder. This way, you can create master template files
 and simply override them on a case-by-case basis within your theme folder.
 
+Now download a new plugin for your theme. The plugin is loaded in **config/bootstrap.php**.
```
Comment (markstory) [inline en/views/themes.rst]: Where is the reader downloading the theme from?

Comment (patsuckow) [reply en/views/themes.rst]: Yes

Comment (patsuckow) [reply en/views/themes.rst]: Do you understand my idea?

Comment (markstory) [reply en/views/themes.rst]: Yes. I'll tweak this a bit.

**Pull request merged**